docs/examples/ephiperfifo.c

```diff
@@ -206,8 +209,8 @@ static void event_cb(GlobalInfo *g, int fd, int revents)
   CURLMcode rc;
   struct itimerspec its;
 
-  int action = (revents & EPOLLIN ? CURL_POLL_IN : 0) |
-               (revents & EPOLLOUT ? CURL_POLL_OUT : 0);
+  int action = (revents & EPOLLIN ? CURL_CSELECT_IN : 0) |
+               (revents & EPOLLOUT ? CURL_CSELECT_OUT : 0);
```

Comment:
The [curl_multi_socket_action man page](https://curl.haxx.se/libcurl/c/curl_multi_socket_action.html) says the `CURL_POLL_*` defines should be used (but they have the same values as the `*CSELECT*` ones).

Reply:
I'm confused now, as in this very man page I read:

> When the events on a socket are known, they can be passed as an events bitmask ev_bitmask by first setting ev_bitmask to 0, and then adding using bitwise OR (|) any combination of events to be chosen from CURL_CSELECT_IN, CURL_CSELECT_OUT or CURL_CSELECT_ERR. 

This is also based on the code `hiperfifo.c` which sets `_CSELECT_`.

Reply:
Oh, you're right. So I should first clarify that... =)

Reply:
I was wrong! I looked at the callback, not the function. My bad.

Reply:
I just did the same thing and got confused myself. Presumably this is the second time I made that mistake. The first being when I wrote it!